lib/Demangling/Demangler.cpp

```diff
@@ -1729,6 +1729,12 @@ NodePointer Demangler::demangleImplFunctionType() {
   if (nextIf('e'))
     type->addChild(createNode(Node::Kind::ImplEscaping), *this);
 
+  // SWIFT_ENABLE_TENSORFLOW
+  if (nextIf('d'))
+    type->addChild(createNode(Node::Kind::ImplDifferentiable), *this);
+  if (nextIf('l'))
+    type->addChild(createNode(Node::Kind::ImplLinear), *this);
+
```

Comment:
Swift enable tensorflow end?

Reply:
There's no strict rule for `// SWIFT_ENABLE_TENSORFLOW END`, I add it only when I deem the "end of added code" to be ambiguous.